data/templated/example.xml

```diff
@@ -97,9 +97,9 @@
          <value>1</value>
          <value>1</value>
          <value>1</value>
-         <value>2022-10-10 11:30:30</value>
-         <value>1.0</value>
-         <value>Test</value>
+         <value dist="uniform" min="1674477220000" max="1706013220000"/>
```

Comment:
Why no test for distributions of timestamp types?  That was supported, right?

Reply:
Rather, what's the unit on this min/max?  Can we not parse actual timestamps?

Reply:
I tried to find a clean and lightweight solution but could not come up with something satisfying. Since the templated value has no information about the type, it would require a nasty try-catch solution. Also, different time formats would have to be supported if one wants to do it right. **For now, I suggest to use UNIX time in ms as a common ground**.

In the future, I could also imagine to add a new distribution called "CurrentTime" or something like that, that always uses the current UTC and is only available for Timestamp/Date/Time types. This would make it easier for this specific use-case.

Reply:
Not sure I follow this comment.
The type of the field is known from the `<types>` section, right?
A small try/catch during config load time (not query runtime) to convert them into timestamps seems pretty straight forward to me.

```java
int get_timestamp(String str) {
 int ts;
  try {
    ts = Integer.parseInt(param_value.min)
  }
  catch (NumberFormatException ex) {
    try {
      ts = DateTimeFormatter.parse(param_value.min).getTime()
    }
    catch (DateTimeParseException ex2) {
      // parse error - report to user that their config is malformed
      throw ex;
    }
  }
  return min_ts;
}

if (param_type == TIMESTAMP) {
  int min_ts = get_timestamp(param_value.min);
  int max_ts = get_timestamp(param_value.max);
}
```

No?

Reply:
Added type information and added DateTimeFormatter. The specific format for timestamps is (yyyy-MM-dd HH:mm:ss)